### PR TITLE
Support E2SM_RC v1.03

### PR DESCRIPTION
**Pull request by jing1228**
Support E2SM_RC v1.03

### CODE REVIEW

File: build/bin/build-deps
Comment (eroshiva) [inline build/bin/build-deps:13]: I won't change the build process. Instead, I'd revisit onos-e2t and updated onos-api version there, so it don't cause any issues in the build process.

File: servicemodels/e2sm_rc/v1/e2sm-rc-ies/e2sm_rc.pb.go
Comment (eroshiva) [inline servicemodels/e2sm_rc/v1/e2sm-rc-ies/e2sm_rc.pb.go:9]: Same as in previous PR - please exclude license header and SM-related information (first 8 lines) - it will crash linters and make the build fail.

File: servicemodels/e2sm_rc/v1/e2sm_rc.proto
Comment (eroshiva) [inline servicemodels/e2sm_rc/v1/e2sm_rc.proto:389]: I don't see here a change for 'RanparameterStructure' message - in the previous PR its only item was marked as optional (which corresponds to the specification). Can you please include this change here as well?

File: servicemodels/test_sm_aper_go_lib/v1/test-sm-ies/test_sm.pb.go
Comment (eroshiva) [inline servicemodels/test_sm_aper_go_lib/v1/test-sm-ies/test_sm.pb.go:9]: Same as in previous PR - is this file included in this PR by intent or it's just a leftover? In case of the latter, please exclude it from the PR.

**Review by eroshiva — CHANGES_REQUESTED** [review]
Great job! Please address my minor comments and we are good to go.
I'll put 'Do Not Merge' label to avoid merging this PR until the onos-api version in onos-e2t is bumped to correspond to the latest one (to avoid build problems).

Once PR #381 is merged, please bump the VERSION file in this PR and then we're good to go :)

**Review by eroshiva — CHANGES_REQUESTED** [review]
LGTM - please bump onos-api version in the rest of the SMs and wait until PR #381 (this is the only reason why I'm putting 'Request changes') come in (after that please bump VERSION file)

**Comment by jing1228** [comment]
@woojoong88 to make "make images" passed, "echo "replace github.com/onosproject/onos-api/go  => github.com/onosproject/onos-api/go v0.10.16" >> build/_input/$1/go.mod" should be added to "build/bin/build-deps" before last line. However it can not be uploaded to the PR. With it, "make images" passed in my local workspace.

**Review by eroshiva — APPROVED** [review]
Please bump VERSION file as a part of this PR

**Pull request merged**
